camel-spring-boot-upgrade-recipes/src/main/resources/META-INF/rewrite/4.13.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.apache.camel.upgrade.camel413.CamelSpringBootMigrationRecipe
+displayName: Migrates Camel Spring Boot applications to Camel Spring Boot 4.12
+description: Migrates Camel Spring Boot applications to Camel Spring Boot 4.12
+recipeList:
+  - org.apache.camel.upgrade.camel412.CamelSpringBootMigrationRecipe
```

Comment:
This is not enough, the display name and descrption has to be updated, moreover, it should reference both org.apache.camel.upgrade.camel412.CamelSpringBootMigrationRecipe and camel413 you just created, right?

Reply:
yes, exactly, if CQ == Camel